### ISSUE TITLE
JAWS screen reader fails to announce tooltip of Select element with required attribute.

### ISSUE THREAD

**Issue by AH106586Harika** [FreedomScientific/standards-support#735]
### Summary

JAWS screen reader fails to announce tooltip of Select element with required attribute.
> 
> Example:
> 1. Go to https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_select_required
> 2. Tab till select field to focus on the field.
> 3. It displays a tooltip.

<img width="1380" alt="image" src="https://github.com/FreedomScientific/standards-support/assets/126655256/ec984ff7-5f65-4133-a2e6-e212c7f643b9">

### Expected result

JAWS should announce the tooltip for required fields when the focus is on the field.

### Actual result

JAWS does not announce the tooltip for required fields when the focus is on the field.

#### JAWS version and build number

JAWS 2023.2306.38

 
#### Operating System and version
Windows 10 Enterprise

#### Browser and version:
Edge Browser Version 115.0.1901.188 (Official build) (64-bit)

**Comment by stevefaulkner** [comment]
@AH106586Harika This is not an issue with JAWS it is an issue with the browser. The tooltip content is not exposed in the accessibility tree and therefore is not available to JAWS or any other screen reader

**Comment by JAWS-test** [comment]
The output of the tooltip by the screen reader is also unnecessary, as the tooltip only visually represents the required attribute. However, the required attribute is correctly displayed in the accessibility tree and output by JAWS. Thus all information is available

**Comment by JAWS-test** [comment]
@stevefaulkner I think the badges "browser bug" can be removed
